src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+export const BOOK_ACTION = {
+  ADD: 'add_book',
+  REMOVE: 'remove_book',
+};
+
+const initialState = {
+  books: [],
+};
+
+export default function bookReducer(state = initialState, action) {
+  switch (action.type) {
+    case BOOK_ACTION.ADD:
+      state = {
+        ...state,
+        books: [...state.books, action.payload.book],
+      };
+      return state;
+    case BOOK_ACTION.REMOVE:
+      state = {
+        ...state,
+        books: state.books.filter((book) => book.id !== action.payload.bookId),
+      };
+      return state;
+    default:
+      return state;
+  }
+}
```

Comment:
- [ ] [Optional] Same here, You using the old way to create a redux reducer in the `redux toolkit` you suppose to use `createSlice` to [create a Redux State Slice](https://redux-toolkit.js.org/tutorials/quick-start#create-a-redux-state-slice).